internal/logger/default_logger.go

```diff
@@ -44,43 +44,39 @@ const (
 // If loggerConfig.SetLogger() method is called, the LoggingLevel property will not be used.
 type DefaultLogger struct {
 	*log.Logger
-	Level logger.Level
+	Level int
 }
 
 // New returns a Default Logger with defaultLogLevel.
 func New() *DefaultLogger {
+	numericLevel, _ := logger.GetLogLevel(defaultLogLevel) // defaultLogLevel exists, no err
 	return &DefaultLogger{
 		Logger: log.New(os.Stderr, "", log.LstdFlags),
-		Level:  defaultLogLevel,
+		Level:  numericLevel,
 	}
 }
```

Comment:
This function can just call `NewWithLevel` with the default log level:
```
return NewWithLevel(defaultLogLevel)
```

Reply:
Thanks, will fix that